src/components/message/styledMessages.js

```diff
@@ -33,6 +34,10 @@ export const Reply = styled.h1`
   }
 `;
 
+export const Section = styled.div`
```

Comment:
Looking at the max-width, looks good, fits the design, well done. My complaint, and this is directed at the designer, is that I wish that there was an avatar for the asker side. If you zoom out all the way, the chat bubbles almost look off center becuase the center-line of the conversation container doesn't bisect the left and right edges of the chat-bubbles, but rather the left side of the left avatar and the right side of the right chat bubble. Again, it looks good, it's 100% accurate to the design, just a design complaint :(

Reply:
it might be a mobile best practice not to show the senders avatar bc of screen real estate and the sender knowing who they are, even if it’s symetrical.. if you check out whatsapp, instagram, or twitter dms it's the same way